analyzers/its/expected/akka.net/Akka.Persistence--netstandard2.0-S3254.json

```diff
@@ -4,6 +4,19 @@
 "id":  "S3254",
 "message":  "Remove this default value assigned to parameter 'default'.",
 "location":  {
+"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/akka.net/src/core/Akka.Persistence/Eventsourced.Recovery.cs#L67",
```

Comment:
TP (default value argument followed by a not default value argument):
https://github.com/SonarSource/sonar-dotnet/blob/21bb28658d6f01414af6fe37f95cd5b8fdc22373/analyzers/its/sources/akka.net/src/core/Akka.Persistence/Eventsourced.Recovery.cs#L67